### PR TITLE
Rover: add reefmaster page

### PR DESCRIPTION
**Pull request by rmackay9**
Reefmaster is a popular tool for creating 2D and 3D maps from sonar data (aka bathymetry).  I always forget exactly how to do the import so hopefully this will help me and others.

By the way, AP is really getting popular for bathymetry.  I see it turning up in videos all the time recently.

### CODE REVIEW

File: common/source/docs/common-underwater-sonars-landingpage.rst [hunk 1]
```diff
@@ -16,3 +16,10 @@ Follow the links below (or in the sidebar) for configuration information based u
     Kogger Sonar <common-kogger-sonar>
     Underwater Sonar (Analog) <common-underwater-sonar-analog>
     Triducer <common-underwater-triducer>
+
+[site wiki="rover"]
+Bathymetry Tools
+----------------
+
```
Comment (Hwurzburg) [inline common/source/docs/common-underwater-sonars-landingpage.rst:23]:
```suggestion

.. toctree::
    :maxdepth: 1

    ReefMaster <reefmaster-for-bathymetry>

```

File: common/source/docs/common-underwater-sonars-landingpage.rst [hunk 2]
```diff
@@ -16,3 +16,10 @@ Follow the links below (or in the sidebar) for configuration information based u
...
+ - :ref:`ReefMaster <reefmaster-for-bathymetry>`
```
Comment (Hwurzburg) [inline common/source/docs/common-underwater-sonars-landingpage.rst:24]: remove

File: common/source/docs/common-underwater-sonars-landingpage.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-underwater-sonars-landingpage.rst:23]: fixes build error due to not being in a TOC tree

File: common/source/docs/common-underwater-sonars-landingpage.rst (hunk 2 above)
Comment (rmackay9) [reply common/source/docs/common-underwater-sonars-landingpage.rst]: removed thanks.

**Comment by rmackay9** [comment]
@Hwurzburg thanks for the review, I've fixed up those issues and it's looking good to me.

**Comment by rmackay9** [comment]
Thanks!

**Pull request merged**